### PR TITLE
update flow

### PR DESCRIPTION
**Pull request by beatrizmaselli**
#### What problem is this solving?

The version 3.x is not yet deployed and there are some errors at the initial flow. We're doing an improvement at the docs and the flow for better experience of the user!

#### How to test it?

[Workspace](https://bibi.myvtex.com/admin)

### CODE REVIEW

**Comment by beatrizmaselli** [comment]
image

File: docs/README.md [hunk 1]
```diff
@@ -11,49 +11,233 @@ You should follow the documentation to [Setting up Cart Abandonment (Trigger)](h
 - The URL field is https://{{account}}.myvtex.com/_v/abandoned-cart.
 - Method field is POST.
 - Header fields:
-  _ content-type: application/json
-  _ accept: application/json
+  content-type: application/json
+  accept: application/json
```
Comment (ghost) [inline docs/README.md:16]: content-type e accept ficaram na mesma linha

File: node/configs/configEmail.ts [hunk 2]
```diff
@@ -14,9 +14,8 @@ export default {
...
+      Subject: 'Lost Cart!',
+      Message:'Template Example',
```
Comment (matcastro) [inline node/configs/configEmail.ts]:
```suggestion
      Message: 'Template Example',
```

**Review by matcastro — COMMENTED** [review]
Just a space character

**Pull request merged**